assignments/pr_review_tutorial/ingram/simple_functions.py

```diff
@@ -1,7 +1,7 @@
 def fibonacci(max):
     values = [0, 1]
-    while values[-2] + values[-1] < max:
-        values.append(values[-2] + values[-1])
+    while values[0] + values[1] < max:
+        values.append(values[0] + values[1])
```

Comment:
This change does not work. This will not result in a Fibonacci sequence as values[0] and values[1] will stay static despite what is appended. This is an infinite loop.